src/components/EditorMarkdownIt.vue

```diff
@@ -143,6 +145,13 @@ export default {
 				}
 			}
 		},
+
+		setSingleCodeRule() {
+			this.md.renderer.rules.code_inline = function(tokens, idx, options, env, self) {
```

Comment:
Is there any reason we don't use (IMHO) more readable arrow functions? Some Nextcloud codinf style guide or some transpiler limitation?

Reply:
I did that because MarkdownIT had an example that i copied, but no technical reason. I will experiment with changing that!